tools/staticbuild/build.sh

```diff
@@ -59,7 +59,11 @@ else
     export PKG_CONFIG_PATH=$DEPS_PATH/lib/pkgconfig:$DEPS_PATH/lib64/pkgconfig:$DEPS_PATH/lib/x86_64-linux-gnu/pkgconfig:$PKG_CONFIG_PATH
 fi
 export FC="gfortran"
-export CPATH=$DEPS_PATH/include:$CPATH
+if [[ $ARCH == 'aarch64' ]]; then
+    export CPATH=/opt/arm/armpl_21.0_gcc-8.2/include_lp64_mp:$CPATH
```

Comment:
If the user installed a different armpl package then this will not work, is there a way to automatically detect the armpl path?

Reply:
does it work with other armpl versions? didn't we see an error when we were using gcc-10 armpl?

Reply:
we can ask users to specifically install `armpl_21.0_gcc-8.2` before installing mxnet wheel